imdclient/tests/minimalReader.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import select
+import time
+from imdclient.IMDClient import IMDClient
+from .utils import parse_host_port
+
+logger = logging.getLogger("imdclient.IMDClient")
+
+
+class minimalReader:
+    """
+    Minimal reader for testing purposes
+
+    Parameters
+    ----------
+    filename : str
+        a string of the form "host:port" where host is the hostname
+        or IP address of the listening MD engine server and port
+        is the port number.
+    n_atoms : int (optional)
+        number of atoms in the system. defaults to number of atoms
+        in the topology. don't set this unless you know what you're doing.
+    kwargs : dict (optional)
+        keyword arguments passed to the constructed :class:`IMDClient`
+    """
+
+    def __init__(self, filename, n_atoms=None, process_stream=False, **kwargs):
```

Comment:
If you don't use `n_atoms` in tests, remove it.

If `process_stream` is always `True` in the tests, remove it.

Keep the minimalReader really minimalistic and not any more complicated than it needs to be.